backend/bot/replies/module/vi.json

```diff
@@ -0,0 +1,27 @@
+{
+  "not_on_a_server": "Lệnh này chỉ có thể chạy trên một máy chủ.",
+  "disabled_command": "Lệnh này đã bị vô hiệu.",
+  "change_nickname_forbidden": "Xin lỗi, tôi không thể đổi biệt danh của bạn. Có thể bạn đang có quyền lớn hơn bot, hoặc bot không có quyền `Quản lý biệt danh`. Vui lòng liên hệ quản trị viên.",
+  "change_role_forbidden": "Xin lỗi, tôi không thể thêm vai trò cho bạn. Bot không có quyền `Quản lý vai trò`. Vui lòng liên hệ quản trị viên.",
+  "restricted_dm": "Tôi không thể nhắn tin trực tiếp cho bạn để khởi động quá trình liên kết. Hãy tạm thời cho phép tin nhắn trực tiếp từ các thành viên máy chủ.",
+  "lack_permission": "Có vẻ như tôi thiếu vài quyền để thực hiện lệnh này, nhiều khả năng là do tôi không có quyền gửi tin nhắn tới kênh đó. Nếu bạn không phải là một quản trị viên, vui lòng liên hệ quản trị viên.",
+  "unknown_error": "Ôi không, có chuyện chẳng lành rồi. Hãy liên hệ cho tôi trên Discord (`Spartan Plume#5172`).",
+  "not_guild_owner": "Bạn cần là chủ của máy chủ để dùng lệnh này.",
+  "no_rights": "Bạn không có quyền để dùng lệnh này.",
+  "invalid_role_name": "Xin lỗi, `%1` là tên vai trò không hợp lệ trên Discord. Vui lòng liên hệ quản trị viên.",
+  "role_does_not_exist": "Vai trò `%1` không tồn tại. Nhờ quản trị viên tạo vai trò đó trước khi dùng lệnh này.",
+  "not_required_role": "Bạn cần vai trò `%1` để dùng lệnh này.",
+  "user_not_found": "Không tìm thấy người dùng `%1`.",
+  "not_linked": "Tài khoản chưa được liên kết. Vui lòng dùng lệnh `%0link`.",
+  "not_verified": "Tài khoản chưa được xác minh. Vui lòng làm theo hướng dẫn được gửi tới bạn sau khi dùng lệnh `%0link`.",
+  "osu_error": "Ôi không, có lỗi vừa xảy ra. Vui lòng thử lại sau. Nếu vấn đề tái diễn, liên hệ tôi trên Discord (`Spartan Plume#5172`).",
+  "tosurnament_api_error": "Một lỗi xảy ra:\nMã: `%1`\nMô tả: `%2`",
+  "usage": "Sử dụng",
+  "example": "Ví dụ",
+  "none": "Không có",
+  "match": "Trận đấu",
+  "referee": "Trọng tài",
+  "streamer": "Người phát trực tiếp",
+  "commentator": "Bình luận viên",
+  "undefined": "Không xác định"
+}
```

Comment:
2 new fields have been added (used in get_matches and show_next_matches commands): `qualifier` and `teams`